frontend/src/static/js/main.js

```diff
@@ -189,17 +189,37 @@ window.app = new window.Vue({
     deactivateTab() {
       this.isTabActive = false;
       window.addHash('tabOpen', this.isTabActive);
-      window.removeHash('tabAuthor');
-      window.removeHash('tabRepo');
       window.removeHash('tabType');
+      this.removeZoomHashes();
+      this.removeAuthorshipHashes();
       window.encodeHash();
     },
 
+    removeAuthorshipHashes() {
+      window.removeHash('tabAuthor');
+      window.removeHash('tabRepo');
+    },
+
+    removeZoomHashes() {
+      window.removeHash('zS');
```

Comment:
After this PR, can you put up an issue to combine all the zoom hashes into a single hashes using base64 encoding?
This is to centralize the use of aliases, to limit the appearance of e.g. `zU` in our code

Reply:
Yes will do.